DTS-STN/passport-status
# Update timeout modal

## [ADO-1585](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/1585)

### Description
This PR updates the idle timer functionality to prompt the user after 10 minutes of inactivity and after an additional 5 minutes of inactivity they are redirected back to the `/expectations` page.

After the `timeout` value of `useIdleTimer` has expired, `onPrompt` is called which opens the modal and starts the timer specified in `promptTimeout`. Once that has expired, `onIdle` is called which forcefully redirects the user.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Set the value of `timeout` to `15 * 20 * 100` and `promptTimeout` to `15 * 10 * 100` or something similar
4. Ensure that after 30 seconds the modal pops up notifying you of your idle state and that you will be redirected
5. Ensure that after 15 seconds you are directed to the `expectations` page
